src/GrassPokemon.java

```diff
@@ -0,0 +1,51 @@
+package src;
+
+public class GrassPokemon extends Pokemon{
+    GrassPokemon(String name, String foodName, String sound) {
+        super.setName(name);
```

Comment:
Instead of calling the super.set... function for each attribute, you could just call the constructor from Pokemon with `super(name, foodName, sound)`